SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/WorkerViewTest.cpp

```diff
@@ -24,7 +24,7 @@ Worker_Op CreateEmptyCreateEntityResponseOp()
 WORKERVIEW_TEST(GIVEN_WorkerView_with_one_CreateEntityRequest_WHEN_FlushLocalChanges_called_THEN_one_CreateEntityRequest_returned)
 {
 	// GIVEN
-	WorkerView View;
+	WorkerView View(nullptr);
```

Comment:
Nit pick that runs throughout - for primitive and (pesudo) pod types I think it's more common to set stuff with an equals than a constructor.